bokehjs/src/lib/models/glyphs/webgl/single_marker.ts

```diff
@@ -58,6 +58,6 @@ export abstract class SingleMarkerGL extends BaseMarkerGL {
     }
     this._show.update()
 
-    this._draw_one_marker_type(marker_type, transform, main_gl_glyph)
+    this._draw_one_marker_type(main_gl_glyph.marker_type, transform, main_gl_glyph)
```

Comment:
Note the `marker_type` is now determined after `set_data` is called as that is where `_border_radius_nonzero` is set.